src/renderer/components/hotbar/hotbar-menu.tsx

```diff
@@ -157,6 +157,7 @@ export class HotbarMenu extends React.Component<Props> {
                             className={cssNames({ isDragging: snapshot.isDragging })}
                             remove={this.removeItem}
                             add={this.addItem}
+                            size={40}
```

Comment:
Shouldn't it default to width? Or would that be too complicated?

Reply:
Maybe, currently it does not... this PR just made the fixed width adjustable through props. If we want this to default to container width it should be done in another PR.

Reply:
Sounds good